sympy/geometry/ellipse.py

```diff
@@ -1459,6 +1459,8 @@ def section_modulus(self, point=None):
         >>> e = Ellipse(Point2D(0, 0), 2, 4)
         >>> e.section_modulus()
         (8*pi, 4*pi)
+        >>> e.section_modulus((2, 2))
+        (2*pi, 2*pi)
```

Comment:
```suggestion
        (16*pi, 4*pi)
```

Reply:
This is not a typo, a e.section_modulus(Point(2,2)) returns (2*pi, 2*pi), naturally e.section_modulus((2,2)) should return the same, it passes the test on local but not on Travis CI.

Reply:
I get this:
```
>>> e = Ellipse(Point(0, 0), 2, 4)
>>> e.section_modulus()
(8*pi, 4*pi)
>>> e.section_modulus((2, 2))
(16*pi, 4*pi)
>>> 
```

Reply:
travis' log:
>Failed example:
    e.section_modulus((2, 2))
Expected:
    (2*pi, 2*pi)
Got:
    (16*pi, 4*pi)

`(2*pi, 2*pi)` is the correct output for `Ellipse(Point(0, 0), 2, 2)`, but for `Ellipse(Point(0, 0), 2, 4)`, it should be `(16*pi, 4*pi)`.

Reply:
That is correct, thank you.